cmd/bridge/config.go

```diff
@@ -55,6 +55,8 @@ type Auth struct {
 type Customization struct {
 	Branding             string `yaml:"branding"`
 	DocumentationBaseURL string `yaml:"documentationBaseURL"`
+	CustomProductName    string `yaml:"customProductName"`
+	CustomLogoFile       string `yaml:"customLogoFile"`
```

Comment:
Definitely we should describe the intent here, that it is expected the image will be mounted via configmap/volume mount in the container.  Its not `base64`, which is typically represented as a `string`.